eos-tech-support/eos-list-users

```diff
@@ -1,27 +1,41 @@
 #!/bin/bash -e
 
-# Lists all users (i.e., all home directories on /dev/sda2)
-
-DEVICE=/dev/sda2
-MOUNT=/mnt/sda2
+# Lists all users (i.e., all home directories on all ostree devices)
 
 USERID=$(id -u)
 if [ "$USERID" != "0" ]; then
     echo "Program requires superuser privileges"
     exit 1
 fi
 
+MOUNT=`mktemp -d`
+
 cleanup() {
     set +e
-    umount $MOUNT
+    if mount | grep $MOUNT; then
+        umount $MOUNT
+    fi
     rmdir $MOUNT
 }
 trap cleanup EXIT
 
 mkdir -p $MOUNT
-mount $DEVICE $MOUNT
 
-# FIXME A more robust approach would be the following:
-# - read /usr/etc/login.defs and extract UID_MIN
-# - read /etc/passwd and get all users with uid >= UID_MIN
-ls $MOUNT/home
+DEVICES=`blkid -t LABEL=ostree | awk -F ":" '{print $1}'`
```

Comment:
When running this on my dev-converted laptop, it looks like it printed two devices with the label:
```
/dev/block/8
/dev/sda3
```

Did you verify it does not happen on a regular system? I guess it's OK to list them both, but it may be a bit confusing if they actually are the same.